### ISSUE TITLE
code Contact Form

### ISSUE THREAD

**Issue by anitadesigns** [hackforla/expunge-assist#1213]

**Comment by CBx3000** [comment]
The Contact Form can use the email expungeassist@hackforla.org

**Comment by jyehllow** [comment]
@sydneywalcoff tagging just to make sure everything's in order for handing off to dev

**Comment by sydneywalcoff** [comment]
@jyehllow no, thank you! it looks good. I just reviewed it and added to the backlog!

**Comment by eunicode** [comment]
ETC: 9/12/2025

**Comment by rcpeng34** [comment]
@sydneywalcoff saw that Eun has dev tickets under this in progress. Can you let me know when it's merged and we'll close out this parent item? thx
